# libdb_cxx-X.Y.so already includes all functions/variables from
# libdb-X.Y.so.  Having the same functionality twice in address space
# means ~ .5MB bigger RAM usage and about 650 prelink conflicts.

--- oo_1.1_src/xmlhelp/util/makefile.mk.jj	2003-04-28 18:18:57.000000000 +0200
+++ oo_1.1_src/xmlhelp/util/makefile.mk	2003-10-21 15:10:35.000000000 +0200
@@ -105,7 +105,6 @@ SHL1STDLIBS=                     \
 	$(SABLOT3RDLIB)          \
 	$(EXPATASCII3RDLIB)      \
 	$(UCBHELPERLIB)          \
-	$(BERKELEYLIB)           \
 	$(BERKELEYCPPLIB)
 
 SHL1LIBS =                       \
